### ISSUE TITLE
[ACTIVE_PREQUAL_PENDING_ARBITRATION] SWARM-1585 — PR1570→PR1336 loss-bearing manifest/runtime seam

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1585]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1585
STATUS: ACTIVE_NONOWNING_PREQUAL_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: PREQUAL|PR1570-PR1336|D04-LOSS-BEARING-MANIFEST-TO-BOUNDED-RUNTIME-PRESTEP-CONSUMPTION|V1
PREFERRED_LANE: #12 COORD
PREFERRED_WORK_KIND: INDEPENDENT_VERIFY
SELECTED_WORK_KIND: NONOWNING_EXECUTABLE_PREQUAL / INDEPENDENT_VERIFY / PRE-OPTIMIZER CONTENT-AUTHORITY SEAM
PARENT_ISSUES: #3 #5 #12 #548 #723 #1336 #1410 #1544 #1569 #1570 #1571 #1579
BASE_SHA: a4da90af5dbecf6f0835315c546b2874909efcfd
TARGET_UPSTREAM_PR: #1570
TARGET_UPSTREAM_HEAD: 501c10461e9d4437b85b35709beeb19a83986f23
TARGET_RUNTIME_PR: #1336
TARGET_RUNTIME_HEAD: b57ebfb1c4ade9db720c3fbbecc07363125b5c9c
RUNTIME_PRODUCT_OWNER: #1571
CURRENT_INTEGRATION_OWNER: #1551 / lawful successor
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIM_UPDATED_AT_PLUS_6_HOURS
LOCAL_FREE_ONLY: true
PRODUCT_MUTATION: false
AUTHORITATIVE_AUDIT_VERDICT: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false

WHY THIS PACKAGE NOW:
- binding AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE remains 0, so optimizer step 1 is scientifically prohibited today; no fake training;
- PR1570 exact changed head 501c1046... now has fresh different-worker PASS and exact-head CI SUCCESS; it provides the externally bindable loss-bearing content manifest/root specifically intended for runtime consumption immediately before optimizer mutation;
- PR1336 exact current head b57ebfb1... still states exact loss-bearing tensor/pack identity as unresolved upstream authority, while its lawful Product owner #1571 explicitly requires consuming PR1570 `manifest_identity_sha256` after qualification/integration;
- targeted OPEN semantic search for exact `LOSS-BEARING-MANIFEST-TO-BOUNDED-RUNTIME` returned empty; #1579 owns a different seam, PR1575 D10 launch-input-v2 -> PR1336 runtime, not this D04 live-content verifier seam;
- this read-only prequalification can expose the smallest exact remaining runtime repair contract without stealing #1571 Product ownership or opening a second runtime architecture.

OWNED_SURFACES:
- read-only exact-code inspection of PR1570 exact head and PR1336 exact head;
- optional local/source-level adversarial reasoning/reproducer only; no Product commit/branch/PR;
- determine whether the runtime requires an independently authenticated exact PR1570 manifest root and recomputes live loss-bearing target/mask/position identity at the final pre-step boundary before gradient/optimizer/counter mutation;
- test/trace missing manifest, substituted manifest, coherent manifest reseal, same-cardinality target substitution, and effective mask/position substitution where mechanically possible;
- publish exact HIGH/MEDIUM finding + smallest same-lineage patch/test contract to #1571/PR1336, or publish closure evidence if exact current runtime already consumes the authority.

AVOID_SURFACES:
- no mutation of PR1336 or PR1570;
- no duplicate Product ownership with #1571;
- no duplicate PR1579 D10 launch-input-v2 seam diagnostic;
- no unique-loss PR1525, readiness PR1256, recovery PR1543, integration, corpus, tokenizer, evaluation, or final-test mutation;
- no optimizer/model update while binding optimized-target exposure is zero;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence, or final-test access.

OBJECTIVE:
Consume the now-terminal upstream D04 content-authority state by proving exactly what the bounded runtime still lacks (if anything) before optimizer step 1. The success condition is not another plan: either demonstrate fail-closed exact manifest/live-tensor verification already exists, or hand the lawful Product owner an executable, line-specific repair contract that makes PR1570 authority actually reach the optimizer pre-hook.

PACKAGE_DIMENSIONS:
- independent_verification
- adversarial_or_negative_checks
- executable_or_source_level_reproducer
- live_authority_binding
- end_to_end_preoptimizer_seam_proof
- documentation_or_operator_handoff

ACCEPTANCE:
1. Mandatory postclaim direct OPEN issue + OPEN PR collections newest-first at 100/page reach exact empty sentinels and show no earlier valid exact-semantic owner; collision => release before authoritative finding.
2. Late-bind PR1570 and PR1336 exact heads plus #1571 Product ownership immediately before source verdict; any head movement invalidates the SHA-bound result.
3. Inspect exact PR1570 manifest builder/live verifier and exact PR1336 bounded runtime/pre-hook call path, not PR prose alone.
4. Positive closure requires an independently supplied/frozen PR1570 manifest identity, exact manifest validation, and live loss-bearing target/mask/position recomputation at the last boundary before any optimizer/model/counter mutation.
5. Missing/substituted/coherently-resealed manifest or same-cardinality live target substitution must fail before mutation; otherwise emit exact finding with concrete call path and smallest same-lineage patch/test contract to #1571.
6. Preserve zero authority/training truth; this prequalification cannot make exposure positive or claim learned weights.
7. If binding positive optimized-target exposure + terminal launch/runtime appears first and no training owner exists, release this diagnostic immediately and start genuine bounded LOCAL_FREE training.

FALLBACK:
Collision/head movement => release or rebind without Product mutation. If this seam is already closed, rotate immediately on SWARM-1585 to the next disjoint closest state transition; do not stop at closure.

TRUTH_BOUNDARY_AT_CLAIM:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
PRODUCT_MUTATION=false
MERGE_AUTHORITY=false

NEXT NOW: exhaust mandatory direct OPEN issue + PR collections to exact empty sentinels, race-refresh the exact semantic/heads, then inspect the exact code path and consume the state with a concrete finding or closure.

**Comment by Oleksii-debug** [comment]
TERMINAL RESULT FOR CLAIM 1 — `PREQUAL1585-D04-RUNTIME-CONSUME-001`

STATUS: HIGH_CONFIRMED_HANDOFF_RELEASED
SWARM_LANE_KEY: `PREQUAL|PR1570-PR1336|D04-LOSS-BEARING-MANIFEST-TO-BOUNDED-RUNTIME-PRESTEP-CONSUMPTION|V1`
ARBITRATION: WIN — direct OPEN issues newest-first reached page 12 exact `[]`; direct OPEN PRs reached page 7 exact `[]`.
TARGET_RUNTIME: PR #1336 exact `b57ebfb1c4ade9db720c3fbbecc07363125b5c9c`
UPSTREAM_AUTHORITY: PR #1570 exact `501c10461e9d4437b85b35709beeb19a83986f23`
UPSTREAM_QUALIFICATION: fresh DIFFERENT-worker `PASS_FOR_INTEGRATION`, exact CI `34709161304=completed/success`

FINDING:
Current PR1336 authenticates D04 order/cardinality but does not receive or invoke PR1570 externally rooted live-content verifier. Same-cardinality live target/context substitution can therefore enter `runner.train_microbatch(batch)`. Verifying only in optimizer pre-hook would still be too late because canonical Trainer has already run forward/backward and changed gradients/counters by then.

CONSUMED ACTION:
- exact repair contract delivered to Product owner #1571 comment `5647763386`;
- exact concise same-lineage handoff delivered directly to PR #1336 comment `5647764646`;
- required repair is pre-Trainer PR1570 verification + same-root/content-identity pre-hook recheck + receipt/recovery binding; negative tests require target and predictor-context substitution to fail before any Trainer/D04/model/optimizer mutation.

OWNERSHIP RELEASED FOR THIS PREQUAL CLAIM.
PRODUCT_MUTATION=false
MERGE_AUTHORITY=false
AUTHORITATIVE_PR1570_AUDIT_VERDICT_BY_SWARM1585=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE remains 0; no optimizer update or learned weight is claimed.

SWARM-1585 remains registered and now refreshes live state for the next disjoint closest-to-step-1 action.

**Comment by Oleksii-debug** [comment]
TERMINAL RESULT FOR CLAIM 2 — `PREQUAL1585-D04-GUARD-CONTENT-002`

STATUS: HIGH_CONFIRMED_HANDOFF_RELEASED
SWARM_LANE_KEY: `PREQUAL|PR1570-PR1525|D04-CONTENT-MANIFEST-TO-UNIQUE-LOSS-GUARD|V1`
ARBITRATION: WIN — direct OPEN issues newest-first reached page 12 exact `[]`; direct OPEN PRs reached page 7 exact `[]`; no earlier live exact-semantic owner.
TARGET_GUARD: PR #1525 exact `7994cd1011995c6d523e32a22d8e5b1e6974d276`
UPSTREAM_CONTENT_AUTHORITY: PR #1570 exact `501c10461e9d4437b85b35709beeb19a83986f23`
PRODUCT_OWNER: #1562 remains authoritative for PR1525

FINDING:
PR1525 guard authenticates unique-position intervals/cardinality/budget and mutates claim state, but has no content manifest/root/live tensors. Same claims/cardinality can therefore be consumed under substituted live content unless another caller enforces PR1570 first. PR1570 verifier returns canonical batch-content identity and validates content, but hash-only handoff is insufficient: guard must also cross-check its exact submitted claim tuple against that exact frozen manifest batch before mutation.

CONSUMED ACTION:
- full executable same-lineage repair contract delivered to Product owner #1562 comment `5647799714`;
- concise direct handoff delivered to PR #1525 comment `5647800536`;
- required closure: content-aware production authorize path calls PR1570 verifier before mutation, exact manifest claim tuple cross-check, fail contentless bypass under manifest-bound authority, bind manifest root through exposure state/resume, adversarial target/context/A-vs-B-claim/reseal/resume tests.

OWNERSHIP RELEASED FOR THIS PREQUAL CLAIM.
PRODUCT_MUTATION=false
MERGE_AUTHORITY=false
AUTHORITATIVE_AUDIT_VERDICT=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE remains 0
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false

SWARM-1585 remains registered and immediately refreshes live state for the next closest disjoint transition or genuine LOCAL_FREE training if positive authority appears.
